<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_fondo">
     <property name="minimumSize">
      <size>
       <width>363</width>
       <height>476</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>363</width>
       <height>476</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(47, 92, 147, 255), stop:1 rgba(26, 0, 31, 255));
border-style: solid;
border-width: 3px;
border-color: rgb(85, 255, 255);
border-radius: 25;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_ayuda">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(0,0,0,0);
border: none;
border-radius: 24;
}
QPushButton:hover{
background-color:rgba(3, 239, 254, 0.4);
font:75 10pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/ayuda 2.0.png</normaloff>../../Downloads/ayuda 2.0.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_salir">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(0,0,0,0);
border: none;
border-radius: 24;
}
QPushButton:hover{
background-color:rgba(3, 239, 254, 0.4);
font:75 10pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/salir 2.1.png</normaloff>../../Downloads/salir 2.1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>400</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-color: rgba(3, 239, 254, 1);
border-style: solid;
border-radius: 10;
border-width: 2px;
background-color: rgba(170, 169, 162, 0);
font:75 10pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
}

QPushButton:hover{
background-color:rgba(3, 239, 254, 0.4);
font:75 10pt &quot;Myanmar Text&quot;;
font: bold;
color: rgb(3,239,254);
}</string>
      </property>
      <property name="text">
       <string>Iniciar sesión</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Downloads/perfil-del-usuario (1).png</normaloff>../../Downloads/perfil-del-usuario (1).png</iconset>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_icono">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../Downloads/circuito-cerrado-de-television.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_user">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0,0,0,0);
border: none;
font:75 14pt &quot;Myanmar Text&quot;;
font: bold;
color: white;</string>
      </property>
      <property name="text">
       <string>Usuario:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_user_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0,0,0,0);
border: none;
font:75 14pt &quot;Myanmar Text&quot;;
font: bold;
color: white;</string>
      </property>
      <property name="text">
       <string>Contraseña:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_user">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
border:none;
font:75 10pt &quot;Myanmar Text&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Ingrese su usuario</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_password">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>350</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;
border:none;
font:75 10pt &quot;Myanmar Text&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Ingrese su contraseña</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_WrongUser">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>320</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));
font: 75 8pt &quot;Arial&quot;;
color: rgb(255, 0, 4);
border: none;
font: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_WrongPswd">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>370</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));
font: 75 8pt &quot;Arial&quot;;
color: rgb(255, 0, 4);
border: none;
font: bold;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
